src/std/make.ss

```diff
@@ -68,6 +101,7 @@
       optimize: (optimize #t) debug: (debug 'env)
       static: (static #t) static-debug: (static-debug #f)
       verbose: (verbose #f) build-deps: (build-deps_ #f)
+      parallelize: (parallelize_ #f)
```

Comment:
I think a saner behaviour is to have `#f` mean "don't parallelize" and `#t` mean "parallelize as much as possible", while integers would specify the number of processes to use (as they currently do).

Reply:
It can default to `#t` obviously.